docs/data/data-grid/column-dimensions/column-dimensions.md

```diff
@@ -58,6 +58,43 @@ To capture changes in the width of a column there are two callbacks that are cal
 - `onColumnResize`: Called while a column is being resized.
 - `onColumnWidthChange`: Called after the width of a column is changed, but not during resizing.
 
+## Autosizing [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+`DataGridPro` also allows for autosizing the columns' dimensions based on their content. It can be autosized by 3 methods:
+
+- Adding the `autosizeOnMount` prop,
+- Double-clicking a column header separator on the grid,
+- Calling the `apiRef.current.autosizeColumns(options)` API method.
+
+You can pass options directly to the API method when you call it. If you need to configure autosize for the other two methods, you can provide the options in the `autosizeOptions` prop. Note that for the separator double-click method, the clicked column will be passed to the `.columns` option, overriding the value of if it exists in `autosizeOptions`. In all cases, the `.minWidth` and `.maxWidth` options of columns will always be respected.
```

Comment:
It will make future git diff easier to handle

```suggestion
You can pass options directly to the API method when you call it. If you need to configure autosize for the other two methods, you can provide the options in the `autosizeOptions` prop.
Note that for the separator double-click method, the clicked column will be passed to the `.columns` option, overriding the value of if it exists in `autosizeOptions`.
In all cases, the `.minWidth` and `.maxWidth` options of columns will always be respected.
```